packages/gigamon/data_stream/ami/manifest.yml

```
@@ -0,0 +1,526 @@
+title: Custom HTTP Endpoint Logs
+type: logs
+streams:
+  - input: http_endpoint
+    template_path: http_endpoint.yml.hbs
+    title: Gigamon Ami logs
+    description: Collect json data from gigamon ami
```

```suggestion
    description: Collect json data from Gigamon AMI via HTTP endpoint. 
```

`json data` seem very generic. If there is any better word(s) to replace `json data`, please do it.